src/chem/monet_aeronet2ioda.py

```diff
@@ -0,0 +1,174 @@
+    )
+    parser.add_argument('-i',
+                        '--input',
+                        help="name of aod550 input file(s)",
+                        type=str,
+                        required=False,
+                        default=None)
+    parser.add_argument('-s',
+                        '--start_date',
+                        help='Start Date to retrieve data. Example "2019-01-01"',
+                        type=str,
+                        required=False,
+                        default=None)
+    parser.add_argument('-e',
+                        '--end_date',
+                        help='End date to retrieve data. Example "2019-01-02"',
+                        type=str,
+                        required=False,
+                        default=None)
+    parser.add_argument('-n',
+                        '--nprocs',
+                        help='number of processors to use',
+                        type=int,
+                        default=1,
+                        required=False)
+    parser.add_argument('-f',
+                        '--frequency',
+                        help='Frequency (in time)  on which to interpolate data to. Example --frequency="H"',
+                        type=str,
+                        default='H',
+                        required=False)
+    parser.add_argument('-p',
+                        '--product',
+                        help='Aeronet Product to Download',
+                        default='AOD15',
+                        type='str',
+                        required=False)
+    parser.add_argument('-w',
+                        '--wavelengths',
+                        nargs='+',
+                        default=[440,470,550,670,870,1020,1240],
+                        required=False)
+    parser.add_argument('-o',
+                        '--output',
+                        help="name of ioda output file",
+                        type=str,
+                        required=True)
+    args = parser.parse_args()
+
+    #get product
+    p = args.product
+
+    # get nprocs
+    nprocs = args.nprocs
+
+    # get frequency
+    freq = args.frequency
+
+    # get output
+    out = args.output
+
+    #get wanted wavelengths
+    wv = args.wavelengths
+
+    # get start date and end date
+    s, e = args.start_date, args.end_date
+
+    #get input file
+    #check if local first
+    if args.input is not None:
+        df = aeronet.add_local(args.input, product=p, freq=freq, interp_to_values=wv)
+    else:
+        if s is None or e is None:
+            try:
+                raise ValueError
+            except:
+                print('If using monetio to download data you must provide the start and end date...')
+        else:
+            dates = pd.date_range(start=s,end=e)
+            df = aeronet.add_data(dates=dates,product=p,freq=freq,interp_to_values=wv)
+
+    nc = aeronet_monet_to_jedi(df,wavelengths=wv)
+    nc.to_netcdf(out)
```

Comment:
The IODA API should be used rather than this dependence on Pandas netCDF write, especially for the in-memory IODA-engines storage that is coming soon from @srherbener